### PR TITLE
Feature/add not found route

### PR DESCRIPTION
**Pull request by tanyushanel**
CPT2107211554-1311x734

### CODE REVIEW

File: exadel-trainee-app/src/app/app-routing.module.ts [hunk 1]
```diff
@@ -38,6 +39,7 @@ const routes: Routes = [
     path: Route.result,
     loadChildren: () => import('./models/result/result.module').then((m) => m.ResultModule),
   },
+  { path: '**', component: NotFoundComponent },
```
[the remark this answers is not in the document]
Comment (tanyushanel) [reply exadel-trainee-app/src/app/app-routing.module.ts:42]: Should move up?

File: exadel-trainee-app/src/app/app-routing.module.ts [hunk 2]
```diff
@@ -38,6 +39,7 @@ const routes: Routes = [
     path: Route.result,
     loadChildren: () => import('./models/result/result.module').then((m) => m.ResultModule),
   },
```
Comment (RomPavel) [inline exadel-trainee-app/src/app/app-routing.module.ts:41]: lets also have smth like 'error' path with same component, so we can easily redirect to it on some errors

File: exadel-trainee-app/src/app/models/not-found/not-found.component.scss [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  background: #fff;
+}
+
+.page_404 img {
+  width: 100%;
+}
+
+.four_zero_four_bg {
+  height: 50vh;
+  background-image: url(https://cdn.dribbble.com/users/285475/screenshots/2083086/dribbble_1.gif);
```
[the remark this answers is not in the document]
Comment (tanyushanel) [reply exadel-trainee-app/src/app/models/not-found/not-found.component.scss:16]: OK

File: exadel-trainee-app/src/app/app-routing.module.ts (hunk 2 above)
Comment (RomPavel) [reply exadel-trainee-app/src/app/app-routing.module.ts:41]: 'error' path should definitely upper then '**'

Comment (tanyushanel) [reply exadel-trainee-app/src/app/app-routing.module.ts:41]: ok

**Review by RomPavel — APPROVED** [review]

**Pull request merged**